projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-with-variants.ts

```diff
@@ -0,0 +1,500 @@
+import {
```

Comment:
There are many functions duplicate with `checkout-as-persistent-user`. Can we only add the new ones?

Reply:
Made the necessary changes